cli/install/install.go

```diff
@@ -222,6 +232,24 @@ func getTTVersions(local bool, distfiles string) ([]version.Version, error) {
 	return versions, nil
 }
 
+// getTTCommits returns all available commits from repository.
+func getCommits(local bool, distfiles string, install string) ([]string, error) {
```

Comment:
Is `install` a program name? getCommits can be used for different purposes. Installation is only one of them.

Reply:
fixed